homeassistant/components/sensor/google_travel_time.py

```diff
@@ -142,7 +142,7 @@ def __init__(self, hass, name, api_key, origin, destination, options):
         else:
             self._destination = destination
 
-        import googlemaps
+        import googlemaps.exceptions
```

Comment:
In this case you need also import `googlemaps.Client`. But like it is now, it should also not trow this error. You are sure that you have no local problem?